source/_docs/z-wave/device-specific.markdown

```diff
@@ -52,6 +52,21 @@ echo -e -n "...turn on/off string from examples above..." | cu -l /dev/zstick -s
 
 You need to disable the on-board Bluetooth since the board requires the use of the hardware UART (and there's only one on the Pi3). You do this by adding the following to the end of `/boot/config.txt`:
 
+For both processes below you will need to insert your SD card into your PC and open the `/boot/config.txt` file with your favorite text editor.
+
+#####The below information is for the Pi 4 specifically
```

Comment:
`The below information` That is the whole point of a paragraph title... ?

We are currently at heading level 3, so this would make level 4, yet, you add a level 5? Please adjust that. 

Lastly, add a space between the last hash character (`#`) and the actual title.

Reply:
Been pulling and using software from github for a while, this will be the first PR I've submitted. So, my apologies for causing any headache.

Reply:
Don't worry m8!

Reply:
As far as the paragraph title portion, does it need to be rephrased? Maybe simplified?

Reply:
Yep. Saying "The information below" is kinda what a heading is for 😉